web/src/admin/applications/wizard/steps/ak-application-wizard-application-step.ts

```diff
@@ -71,7 +72,7 @@ export class ApplicationWizardApplicationStep extends ApplicationWizardStep {
             this.errors.set("name", msg("An application name is required"));
         }
 
-        if (!values.metaLaunchUrl || !URL.canParse(values.metaLaunchUrl)) {
+        if (values.metaLaunchUrl && !URL.canParse(values.metaLaunchUrl)) {
```

Comment:
TODO: This was subtle fix. Circle back on something like an `<ak-url-input>` to handle this sort of check.